core/templates/pages/library-page/library-page.component.html

```diff
@@ -395,7 +395,7 @@ <h2 ng-class="{'active': $ctrl.activeGroupIndex === $index}" class="oppia-librar
       font-size: 1.05em;
     }
   }
-  @media (max-width: 992px) {
+  @media (max-width: 1011px) {
```

Comment:
Hi @atpug22 

I'm still able to reproduce the issue even with your fix. PTAL.
![reproable](https://user-images.githubusercontent.com/11008603/103452155-48140400-4cf2-11eb-917a-e043c0965504.gif)

Also, isn't the search bar supposed to go to the middle of the page before it starts to break?

Reply:
@kevintab95 can you please check the changes in code after taking a pull from this pull request because it is working properly on my pc. 

And Yes the search bar was supposed to go to the middle but recently one more item is added in the navbar i.e. classroom so space is reduced further causing such errors in the search bar.

Reply:
Yep, I can confirm that I checked out your changes locally. Can you try switchting to responsive mode in the dev console and check for the same dimensions in the GIF above?

And yep, I think you're right regarding the space issue.. but we have enabled the classroom page on oppia.org so we should have the search bar go to the middle of the page if it does not have enough space in the navbar. Can you please fix this too?

Reply:
Uploaded recording in a comment below.